### DISCUSSION TITLE
localized

### DISCUSSION THREAD

**Discussion by amavo** [arcemu/arcemu#437] [Ideas]
It would be interesting npc_monstersay_localized create the table for future language translations. There are also other tables that could be included.
- trainer_defs_localized

**Comment by ghost** [comment]
Strange that this was still missing ^^ I would strongly support this ;)

**Comment by MesoX** [comment]
It shouldn't be that hard to implement this.

**Comment by dfighter1985** [comment]
I have converted this to discussion and moved to the ideas section.
